pkg/builder/object_meta.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2019 the Velero contributors.
+Copyright 2020 the Velero contributors.
```

Comment:
Don't move copyright dates forward.  That is the date this code is first developed/copyrighted.  We can add 2019-2020 if we actually make changes that we want to start the clock on (it's kind of silly, copyright goes on practically forever anyhow, but I think we've always done it as the initial year it was created)